src/TwigComponent/src/Twig/ComponentExtension.php

```diff
@@ -84,6 +89,43 @@ public function finishEmbeddedComponentRender(): void
         $this->container->get(ComponentRenderer::class)->finishEmbeddedComponentRender();
     }
 
+    public function cva(array &$context, string $recipeName, array $recipe): void
+    {
+        $key = '__recipe__'.$recipeName;
+
+        $context[$key] = CVA::init($recipe);
+    }
+
+    public function twm(array $context, string $recipeName, array $variants, string $custom = null): string
+    {
+        $key = '__recipe__'.$recipeName;
+
+        if (!isset($context[$key])) {
+            return '';
+        }
+
+        $tw = TailwindMerge::instance();
```

Comment:
`TailwindMerge` should be configurable, because some projects can use a prefix (e.g.: `tw-`) for easily transitioning to Tailwind, and I don't think this will work out-of-the-box.

Reply:
Yes sure this a first draft, to show the ideas, but yeah I have in mind to add configuration and caching